dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -95,13 +96,46 @@ def shell(verbose: bool):
 
 
 @option_verbose
-@main.command()
-def build_ci_image(verbose: bool):
-    """Builds breeze.ci image for breeze.py."""
+@main.command(name='build-ci-image')
+@click.option(
+    '--additional-extras',
+    help='This installs additional extra package while installing airflow in the image.',
+)
+@click.option('-p', '--python', help='Choose your python version')
+@click.option(
+    '--additional-dev-apt-deps', help='Additional apt dev dependencies to use when building the images.'
+)
+@click.option(
+    '--additional-runtime-apt-deps',
+    help='Additional apt runtime dependencies to use when building the images.',
+)
+@click.option(
+    '--additional-python-deps', help='Additional python dependencies to use when building the images.'
+)
+@click.option('--github-repository', help='Choose repository to push/pull image.')
+def build_ci_image(
+    verbose: bool,
+    additional_extras: str,
```

Comment:
I think we should use Optional[str] here more often and set defaults to `None` for all parameters that were "empty" by defaiult in Bash. The reason they were '" was because it is easier in Bash (MacOS) to deal with empty than undefined values, but we have much more freedom now in Python and `None = "not set"` is better approach.

Reply:
By default, if the option is not set then, the value for that parameter is None in click option.
Say example , if I didn't pass any value for additional_extras in build_ci_image command, then the value set for additional_extras is None

Reply:
Yes I will use Optional[str] here more often. It makes sense.

Reply:
Yeah - we use "strict_optional" approach in Airlfow so when we finish bringing back `mypy` it will complain if Optional is not set. And setting = None in this case by default also does not hurt (even if click will do it automatically)..